File: my_tools_pkg/src/ros_log.cpp
#include "rclcpp/rclcpp.hpp"

int main(int argc, char * argv[])
{
  rclcpp::init(argc, argv);
  auto node = rclcpp::Node::make_shared("ros2_logging_cpp");

  RCLCPP_DEBUG(node->get_logger(), "This is a DEBUG message.");
  RCLCPP_INFO(node->get_logger(), "This is an INFO message.");
  RCLCPP_WARN(node->get_logger(), "This is a WARNING message.");
  RCLCPP_ERROR(node->get_logger(), "This is an ERROR message.");
  RCLCPP_FATAL(node->get_logger(), "This is a FATAL message.");

  rclcpp::spin(node);
  rclcpp::shutdown();
  return 0;
}

File: my_tools_pkg/scripts/ros_log.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node

def main(args=None):
    rclpy.init(args=args)

    node = Node("ros2_logging_py")
    node.get_logger().debug("This is a DEBUG message.")
    node.get_logger().info("This is an INFO message.")
    node.get_logger().warn("This is a WARNING message.")
    node.get_logger().error("This is an ERROR message.")
    node.get_logger().fatal("This is a FATAL message.")

    rclpy.spin(node)
    node.destroy_node()
    rclpy.shutdown()

if __name__ == '__main__':
    main()